AnnoyancesFilter/sections/popups.txt

```diff
@@ -98,6 +98,9 @@ sodapdf.com##.be-tooltip
 !
 ! Specific sites
 !
+guardyoureyes.com#$##specialModal { display: none !important; }
```

Comment:
what about
`||guardyoureyes.com/templates/mentalpress/js/subscription.js`

Reply:
Do you check the whole site from top to bottom or what?

Reply:
Is something wrong? This site is unavailable at the moment

Reply:
Nope. You find things which I do not find, you reproduce things which I can't reproduce.